### PR TITLE
Fix addons_updater_notification_content_singular string

### PR DESCRIPTION
**Pull request by Amejia481**
Related https://github.com/mozilla-l10n/android-l10n/pull/183#discussion_r393109985
cc: @flodolo

### CODE REVIEW

**Comment by Amejia481** [comment]
bors r=psymoon

File: components/feature/addons/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -111,8 +111,8 @@
     <string name="mozac_feature_addons_updater_notification_title">%1$s has a new update</string>
     <!-- The content of the notification, this will be shown when an add-on needs new permissions, to be updated. %1$d is the amount of new permissions-->
     <string name="mozac_feature_addons_updater_notification_content" tools:ignore="PluralsCandidate">%1$d new permissions are required</string>
-    <!-- The content of the notification, this will be shown when an add-on needs a new permission-->
-    <string name="mozac_feature_addons_updater_notification_content_singular">A new permissions is required</string>
+    <!-- The content of the notification displayed/shown when an add-on needs a new permission-->
```
Comment (flodolo) [inline components/feature/addons/src/main/res/values/strings.xml]: I mean displayed/shown as an alternative, not to use both (just pick one) ;-)

**Comment by Amejia481** [comment]
boars r-

**Comment by Amejia481** [comment]
bors -

**Comment by Amejia481** [comment]
bors -

**Comment by Amejia481** [comment]
bors -r

File: components/feature/addons/src/main/res/values/strings.xml (hunk 1 above)
Comment (Amejia481) [reply components/feature/addons/src/main/res/values/strings.xml]: 👍

**Comment by Amejia481** [comment]
bors r=psymoon

**Comment by Amejia481** [comment]
bors r=psymoon

**Pull request merged**